### PR TITLE
12562 UI mhr search results text updates

### PR DESCRIPTION
**Pull request by ChasBoucher**
*Issue #:* /bcgov/entity12562

Change text from Search Results to Selection List.  See ticket for details

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/README.md [hunk 1]
```diff
@@ -1,8 +1,7 @@
 # ppr-ui
 
 ## Project setup
-```
-npm install
+```npm install
```
Comment (thorwolpert) [inline ppr-ui/README.md]: We don't normally include style changes in a PR that is a functional change.
It tends to bury the changes amongst other stylistic changes, and those have already made it through a PR cycle.

**Review by thorwolpert — COMMENTED** [review]
Just a comment on our general change approach.

**Comment by ChasBoucher** [comment]
Hi Thor,


Makes sense.


Thanks for the info


Charles


________________________________

**Review by dimak1 — APPROVED** [review]

**Comment by thorwolpert** [comment]
You also don't need to copy/paste the notification email, as it's a copy of what is commented here in the PR thread.

**Review by thorwolpert — APPROVED** [review]
lgtm

**Pull request merged**